metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowPostProcessor.java

```diff
@@ -105,9 +115,36 @@ private void indexPostProcess(AbstractExecutablePlugin<?> indexPlugin, String da
     } else if (indexPlugin instanceof IndexToPublishPlugin) {
       targetIndexingDatabase = ((IndexToPublishPlugin) indexPlugin).getTargetIndexingDatabase();
       targetIndexingEnvironment = ((IndexToPublishPlugin) indexPlugin).getTargetIndexingEnvironment();
-      //Reset depublish status
-      depublishRecordIdDao.markRecordIdsWithDepublicationStatus(datasetId, null,
-          DepublicationStatus.PENDING_DEPUBLICATION, null);
+
+      final long totalRecords = dpsClient.getTotalMetisDatabaseRecords(indexPlugin.getExternalTaskId(),
```

Comment:
the totalRecords is type ```long```
create a ticket for ```getDetailedTaskReportBetweenChunks``` receives an ```int``` if the amount it will fail because of variable type. For this time we are skipping this to get the environment running again.